server/api/src/rsSimpleQuery.cpp

```diff
@@ -42,8 +75,13 @@ rsSimpleQuery( rsComm_t *rsComm, simpleQueryInp_t *simpleQueryInp,
         }
     }
     else {
-        status = rcSimpleQuery( rodsServerHost->conn,
-                                simpleQueryInp, simpleQueryOut );
+        status = procApiRequest(
```

Comment:
I can either turn off the deprecation warning here, or do this to get around having to use the `rc` interface. The question was raised in my mind at this point whether we should also be putting a warning message in the log and/or the `rError` stack?

Reply:
`rError` back to the user seems... helpful?    As long as we're still sending the same return code, their code shouldn't break...

Not convinced logging it on the server is as helpful.

Reply:
After discussion, we want to just turn off the deprecation warning (a la apiTable).

Will add warning message to `rError`.